sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -11,15 +11,15 @@ The [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet)
 
 ### Migrating from Application Insights SDK
 
-If you are currently using the Application Insights SDK and want to migrate to OpenTelemetry, please follow our [migration guide](https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-dotnet-migrate?tabs=console). 
+If you are currently using the Application Insights SDK and want to migrate to OpenTelemetry, please follow our [migration guide](https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-dotnet-migrate?tabs=console).
```

Comment:
Could we move the readme to a follow-up PR? I got some blocking comments for it.

Reply:
I removed the new content. The remaining changes are whitespace cleanups that i don't want to undo :)